themes/bootstrap5/scss/variables.scss

```
@@ -16,3 +16,21 @@ $nav-link-hover-bg:       $gray-lighter !default;
 
 $alert-default-bg: #f5f5f5 !default;
 $alert-default-border-color: darken($alert-default-bg, 7%) !default;
+
+// Sets the visual division of columns in which the facets on home page should be displayed
+$home-facet-container-right-padding: 1rem !default; // defines padding for better visual division
+
+
+// Sets the number of columns in which the facets on home page should be displayed for Desktop and smaller devices
+$home-facet-list-border-right-style: 1px solid $gray-dark !default;
+$home-facet-list-column-count-md: 2 !default; // Defines the maximum number of columns for MD sized devices
+$home-facet-list-column-count-sm: 1 !default; // Defines the maximum number of columns for SM sized devices
+$home-facet-list-column-count-xs: 1 !default; // Defines the maximum number of columns for XS sized devices
+$home-facet-list-column-count: 2 !default; // Defines the maximum number of columns for devices larger than MD
+$home-facet-list-column-fill: auto !default; // if set to 'balance', it will distribute available entries evenly across the max. number of columns, set to 'auto' for distribution that fills first column and places remaining entries in 2nd column etc.
+$home-facet-list-column-gap: 1rem !default; // Defines the gap between columns if more than one
+$home-facet-list-column-height: auto !default; // Defines the height of each column before the column breaks, 1rem should roughly equal one line
+$home-facet-list-column-overflow: clip !default; // Defines what's to be done if there are more entries than fit the columns
+$home-facet-list-left-padding: 0 !default; // overwrites default left padding for uls (2rem)
+$home-facet-list-list-style: none !default;
+$home-facet-list-right-padding: 1rem !default;
```

We should probably add an end-of-file line break here for style consistency:

```suggestion
$home-facet-list-right-padding: 1rem !default;

```

---

just re-submitted with the requested changes ....

---

Thanks! I'll wait and see what others have to say before commenting further myself... but let me know if you need my attention on anything in particular.